src/blockchain_processor.py

```diff
@@ -348,6 +352,11 @@ def add_to_history(self, addr, tx_hash, tx_pos, tx_height):
         self.batch_txio[txo] = addr
 
 
+    def get_masternodes_status(self):
+        return self.dashd('masternode', ['list'])
+
+    def get_proposals_status(self):
+        return self.dashd('mnbudget', ['list'])
```

Comment:
I think it is save to assume that version < 120100  uses `mnbudget show` and `mngovernance list` otherwise